test/Dialect/RTG/Transform/elaboration.mlir

```diff
@@ -0,0 +1,54 @@
+// RUN: circt-opt --rtg-elaborate=seed=0 --split-input-file --verify-diagnostics %s | FileCheck %s
+
+func.func @dummy1(%arg0: i32, %arg1: i32, %arg2: !rtg.set<i32>) -> () {return}
+func.func @dummy2(%arg0: i32) -> () {return}
+func.func @dummy3(%arg0: i64) -> () {return}
+
+// Test the set operations and passing a sequence to another one via argument
+// CHECK-LABEL: rtg.test @setOperations
+rtg.test @setOperations : !rtg.dict<> {
+  // CHECK-NEXT: [[V0:%.+]] = arith.constant 4 : i32
+  // CHECK-NEXT: [[V1:%.+]] = arith.constant 2 : i32
+  // CHECK-NEXT: [[V2:%.+]] = arith.constant 3 : i32
+  // CHECK-NEXT: [[V3:%.+]] = rtg.set_create [[V1]], [[V2]] : i32
+  // CHECK-NEXT: func.call @dummy1([[V0]], [[V1]], [[V3]]) :
+  // CHECK-NEXT: }
+  %0 = arith.constant 2 : i32
+  %1 = arith.constant 3 : i32
+  %2 = arith.constant 4 : i32
+  %3 = arith.constant 5 : i32
+  %set = rtg.set_create %0, %1, %2, %0 : i32
+  %4 = rtg.set_select_random %set : !rtg.set<i32> {rtg.elaboration_custom_seed = 0}
+  %new_set = rtg.set_create %3, %4 : i32
+  %diff = rtg.set_difference %set, %new_set : !rtg.set<i32>
+  %5 = rtg.set_select_random %diff : !rtg.set<i32> {rtg.elaboration_custom_seed = 0}
```

Comment:
Would it make sense to have custom seed values that are distinct from the `seed=0` passed to the `--rtg-elaborate` pass? To make sure that the correct seed value is used?